rust/connlib/tunnel/src/client.rs

```diff
@@ -924,6 +943,8 @@ impl ClientState {
         self.active_cidr_resources
             .iter()
             .map(|(ip, _)| ip)
+            .chain(iter::once(IPV4_TUNNEL.into()))
+            .chain(iter::once(IPV6_TUNNEL.into()))
```

Comment:
Technically, for the SRV/TXT record feature, we don't need to expose these routes as all traffic for them will be generated from within connlib.

I've added them nonetheless because I think it could be a useful feature in general. For example, based on this we could extend our DNS resolver to add a dummy domain for each site, allowing the user to "ping a site". I think we also floated the idea of running an SSH server on the Gateway at some point, for that, we'll also need to route packets directly to the TUN device.

Reply:
It also made it possible to test this PR meaningful because now you can just send ICMPs to the Gateway's TUN device.